misc.php

```diff
@@ -935,9 +939,9 @@
 		}
 		if((int)$limit > 0)
```

Comment:
Yes, it's the same double check like [here](https://github.com/SvePu/mybb/blob/26775de517299b33a28e74ae5e84ea39aa779289/misc.php#L909)